indico/modules/events/static/templates/emails/paper_judgment_notification_email.txt

```diff
@@ -0,0 +1,39 @@
+{% extends 'emails/base.txt' %}
+
+{% macro get_comments() %}
+    {% filter dedent -%}
+        Please see the comments from the reviewing team below:
+
+        "{{ paper.judgment_comment }}"
+    {%- endfilter %}
+{% endmacro %}
+
+{% block subject %}The reviewing process of your paper "{{contribution.title}}" for "{{contribution.event_new.title}}" has finished{% endblock %}
+
+{% block header_recipient -%}
+    {{ paper.submitter.first_name }}
+{%- endblock %}
+
+{% block body -%}
+    {% filter dedent -%}
+        {% if paper.state.name in ('accepted', 'rejected') %}
+            The judge has {{ paper.state.name }} your paper entitled "{{ contribution.title }} (#{{ contribution.friendly_id }})", submitted for "{{ contribution.event_new.title }}".
```

Comment:
I would be a bit less explicit with who did the judgment. Here "the judge" sounds a bit out of context. I would go for something more impersonal along the lines of:

```
After the reviewing process your paper ... has ben accepted.
After the reviewing process your paper ... has ben rejected.
After the reviewing process your paper ... is required to be corrected.
```

Thoughts?